### PR TITLE
Fix func checkIfLoaded in proxy

### PR DESCRIPTION
**Pull request by aoiasd**
related:https://github.com/milvus-io/milvus/issues/18659
Signed-off-by: aoiasd [zhicheng.yue@zilliz.com](mailto:zhicheng.yue@zilliz.com)

### CODE REVIEW

**Comment by aoiasd** [comment]
rerun ut

**Comment by aoiasd** [comment]
rerun ut

**Comment by aoiasd** [comment]
rerun ut

File: internal/proxy/task_search.go [hunk 1]
```diff
@@ -444,52 +444,32 @@ func checkIfLoaded(ctx context.Context, qc types.QueryCoord, collectionName stri
...
 	if resp.Status.ErrorCode != commonpb.ErrorCode_Success {
 		return false, fmt.Errorf("showPartitions failed, collection = %s, partitionIDs = %v, reason = %s", collectionName, searchPartitionIDs, resp.GetStatus().GetReason())
 	}
 
-	if len(resp.GetPartitionIDs()) > 0 {
-		log.Ctx(ctx).Warn("collection not fully loaded, search on these partitions",
-			zap.String("collection", collectionName),
-			zap.Int64("collectionID", info.collID), zap.Int64s("partitionIDs", resp.GetPartitionIDs()))
-		return true, nil
+	for _, persent := range resp.InMemoryPercentages {
```
Comment (congqixia) [inline internal/proxy/task_search.go:467]: NIT: misspell

**Review by congqixia — COMMENTED** [review]
/lgtm
/cc @XuanYang-cn @yah01

**Pull request merged**